__tests__/index.spec.ts

```diff
@@ -119,9 +119,10 @@ describe('use-herald-action', () => {
                 "*.ts",
               ],
               "includesInPatch": Array [],
+              "isMemberOf": Array [],
               "matched": true,
               "name": "rule1.json",
-              "path": "${env.GITHUB_WORKSPACE}/__mocks__/rules/rule1.json",
+              "path": "/Users/gfrigerio/base/use-herald/__mocks__/rules/rule1.json",
```

Comment:
this needs to change back. 
```suggestion
              "path": "${env.GITHUB_WORKSPACE}/__mocks__/rules/rule1.json",
```